docs/src/pages/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import Layout from "@theme/Layout";
+import Features from "../components/homepage/Features";
+import AnimatedText from "../components/homepage/AnimatedText";
+import HowItWorks from "../components/homepage/HowItWorks";
+import Carousel from "../components/homepage/Carousel/Carousel";
+import FeaturesFAQ from "../components/homepage/FeaturesFAQ";
+import CaseStudies from "../components/homepage/CaseStudies";
+import { forLaptop } from "../../helpers/media-queries";
+import useMediaQuery from "../hooks/useMediaQuery";
+import ResponsiveCarousel from "../components/homepage/Carousel/ResponsiveCarousel";
+import HomepageHero from "../components/homepage/HomepageHero";
+
+export default function Home() {
+  const isLaptopUp = useMediaQuery(forLaptop);
+  return (
+    <div className="z-index:1000">
+      <Layout>
+        <HomepageHero />
+        <main>
+          <Features />
+          <div className="pageContainer">
+            <AnimatedText />
```

Comment:
Should: fix the animated text. It runs off-screen:

![image](https://github.com/user-attachments/assets/4bed3ba2-9cd3-4868-a418-4f44b88e52cd)